site/content/integrate/getting-started/how-should-i-integrate.md

```diff
@@ -27,6 +27,10 @@ That's not enough? You can include <a target="_blank" href="https://docs.matterm
 
 To build a richer bot integration, you can make full use of the [Mattermost REST API](/integrate/rest-api/). Everything that you see a Mattermost client doing, your integration can do too with this API.
 
+### I want to develop a cloud-based app 
```

Comment:
Cloud based?

Reply:
I couldn't think of a better way to frame it. How about "I want a more flexible option"?

Reply:
Isn't the selling point the rich UX possibilities?